docker/Dockerfile.cpu

```diff
@@ -53,6 +53,8 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 
 ENV LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal.so.4:/opt/venv/lib/libiomp5.so:$LD_PRELOAD"
 
+ENV TORCHINDUCTOR_COMPILE_THREADS=1
```

Comment:
Why set this to 1?

Reply:
To avoid create too much unused compile threads in background.